browser/brave_ads/ads_service_impl.cc

```diff
@@ -123,7 +123,9 @@ namespace brave_ads {
 
 namespace {
 
-const unsigned int kRetriesCountOnNetworkChange = 1;
+constexpr unsigned int kRetriesCountOnNetworkChange = 1;
+
+constexpr int kNeedToUpdateBrowserReponseCode = 426;
```

Comment:
`net::HTTP_UPGRADE_REQUIRED`

Reply:
there is no such code in `net/http/http_status_code_list.h`. I could rename this constant to `kHttpUpgradeRequiredStatusCode`